polar_route/route_planner.py

```diff
@@ -518,154 +563,57 @@ def compute_routes(self):
-                        break
-
-                    self.nc = nc
-
-                    #nc._mergePoint()
-                    self.nc = nc
-                    iter+=1
-
-                    
-                        
-                    # Stop optimisation if the points are within some minimum difference
-                    if len(nc.previousDF) == len(nc.CrossingDF):
-                        Dist = np.max(np.sqrt((nc.previousDF['cx'].astype(float) - nc.CrossingDF['cx'].astype(float))**2 + (nc.previousDF['cy'].astype(float) - nc.CrossingDF['cy'].astype(float))**2))
-                        self.allDist.append(Dist)
-                        pbar.set_description("Mean Difference = {}".format(Dist))
-                        if (Dist < minimumDiff) and (iter>=minimum_iterations):
-                            logging.info('{} iterations - dDist={}'.format(iter, Dist))
-                            break
-
-                if iter == nc.pathIter:
-                    logging.info('Maximum Iteration Met - Returning Last Path'.format(iter, Dist))
-                    nc.CrossingDF = self.all_crossing_points[-1]
-
-
-
-                # ------ Smooth Path Values -----
-                # Given a smoothed route path now determine the along path parameters.
-                variables      = nc.objective_function(nc.CrossingDF)
-                TravelTimeLegs = variables['traveltime']['path_values']
-                DistanceLegs   = variables['distance']['path_values'] 
-                pathIndex      = variables['cell_index']['path_values'] 
-                FuelLegs       = variables['fuel']['path_values'] 
-                SpeedLegs      = variables['speed']['path_values'] 
-
-
-                # ------ Saving Output in a standard form to be saved ------
-                SmoothedPath ={}
-                SmoothedPath['type'] = 'Feature'
-                SmoothedPath['geometry'] = {}
-                SmoothedPath['geometry']['type'] = "LineString"
-                SmoothedPath['geometry']['coordinates'] = nc.CrossingDF[['cx','cy']].to_numpy().tolist()            
-                SmoothedPath['properties'] = {}
-                SmoothedPath['properties']['from'] = Path['properties']['from']
-                SmoothedPath['properties']['to']   = Path['properties']['to']
-                SmoothedPath['properties']['traveltime'] = list(TravelTimeLegs)
-                SmoothedPath['properties']['fuel']       = list(FuelLegs)
-                SmoothedPath['properties']['distance']   = list(DistanceLegs)
-                SmoothedPath['properties']['speed']      = list(SpeedLegs)
-                SmoothedPaths.append(SmoothedPath)
-                geojson['features'] = SmoothedPaths
-                self.smoothed_paths = geojson
-                self.mesh['paths'] = self.smoothed_paths
-
-                    
\ No newline at end of file
+        paths = []
+        geojson = {}
+        SmoothedPaths = []
```

Comment:
Variables `paths` and `SmoothedPaths` are unused; can they be removed?

Reply:
Paths is unused smoothed_paths is used in next commit